### ISSUE TITLE
v1 : [bug] Cater details shouldn't be allowed to set null

### ISSUE THREAD

**Issue by jey-exp** [jey-exp/Cater#2]
In cater login, while editing cater details, it is allowed to submit with an empty field. 
This updates the details in db as null for that particular field.
Also while submitting with  an empty field, the useState is not updated to null.

- Decide should the cater allowed to enter null
- Or how to handle null, also updation of usestate

**Comment by Oscur007** [comment]
Do you want null to be sent when empty fields are passed from frontend to backend or handle null values differently in backend(which is a backend part)?The message is not clear .

**Comment by anevski-stefan** [comment]
test

**Comment by jey-exp** [comment]
Sorry I didn't expect people to contribute to the project. 

I opened this issue so I don't forget to work on this bug.
If you are interested to work on this project(There will be lot of issues opened soon), please let me know. 
I will add more detailed issues and a contribution guide soon 😄

**Comment by Oscur007** [comment]
Yes I would like to contribute to your project . It would be my pleasure to help .

**Comment by jey-exp** [comment]
Good to hear that! 

Currently this project is small and not well structured, I think it would be good to contribute once I build a proper structure and add a set up guide, as it is now not feasible for you to configure the DB(I haven't planned for others to contribute to it, so didn't add how to configure the DB).

Sorry if this disappointed you, but I make sure that I will ping you once I have fixed all things and raise a new issue!
